spec/course_spec.rb

```diff
@@ -0,0 +1,13 @@
+require ./lib/course.rb
+require ./lib/student.rb
+
+RSpec.describe Course do
+let(:course_data) {:Calculus}
+let(:capacity) {2}
+let(:course) {Course.new}
```

Comment:
Using this syntax may trip you up, if you feel it helps just create the things in each individual test.